app/resources/css-default.template

```diff
@@ -177,11 +177,16 @@ div.languages {
   text-decoration: none;
   color: black;
   text-align: {{end}};
+  float:right;
+  margin-right:5%;
```

Comment:
Thanks. Can you make it 20px padding, to balance with the padding for .title-bar-button?

Also, change padding-right to padding-{{end}}. It is automatically expanded into padding-right for left-to-right languages and padding-left for right-to-left languages.